.github/CODEOWNERS

```diff
@@ -512,6 +512,8 @@
 /core/templates/Polyfills.ts @srijanreddy98
 /extensions/extensions.module.ts @vojtechjelinek
 /schema_utils*.py @DubeySandeep
+/core/controllers/payload_validator.py @Nik-09
```

Comment:
Please suggest names of codeowners for these two files.

Reply:
Please assign @aks681 because he is the codeowner for the entire directory. So any new files will be assigned to him.